graylog-plugin-archetype/pom.xml

```diff
@@ -101,13 +101,12 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-archetype-plugin</artifactId>
-                    <version>2.4</version>
+                    <version>3.0.1</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-resources-plugin</artifactId>
-                    <!-- Using 2.6 instead of 2.7 because of this: https://issues.apache.org/jira/browse/MRESOURCES-190 -->
-                    <version>2.6</version>
+                    <version>3.0.2</version>
```

Comment:
This is still an issue and with version 3.0.2 the archetype JAR does not include the `.gitignore` file while it's included when using version 2.6.

Reply:
✅ Addressed in 5027297983b9edbc96909cbad32f1719c7994b40

Reply:
@joschi Thanks! It would be great if you could add a small comment about why we use that setting. That makes it easier for my future self to understand it. :wink: